src/features/auth/login.ts

```diff
@@ -28,6 +29,12 @@ export const login =
 				if (!matches) {
 					throw new BadRequestError('Current password incorrect.');
 				}
+
+				const userPermissions = await permissions.getUserPermissions(user.id);
```

Comment:
I think that you should be able to replace this w/ a `middleware.permissionRequired('auth.credentials_login', { statusCode: 400, message: 'User not allowed to login.' })`, but
1. we can't do it that way if we want to only make this check after validating the user's password.
2. it's not a big deal
3. we atm do not support that kind of a second argument